enclave/src/main/java/org/hyperledger/besu/enclave/Enclave.java

```diff
@@ -54,42 +56,72 @@ public boolean upCheck() {
     }
   }
 
-  public SendResponse send(final SendRequest content) {
+  public SendResponse sendLegacy(
+      final String payload, final String privateFrom, final List<String> privateFor) {
+    final SendRequestLegacy sendRequestLegacy =
+        new SendRequestLegacy(payload, privateFrom, privateFor);
     return post(
         JSON,
-        content,
+        sendRequestLegacy,
         "/send",
         (statusCode, body) -> handleJsonResponse(statusCode, body, SendResponse.class));
   }
 
-  public ReceiveResponse receive(final ReceiveRequest content) {
+  public SendResponse sendBesu(
+      final String payload, final String privateFrom, final String privacyGroupId) {
+    final SendRequestBesu sendRequestBesu =
+        new SendRequestBesu(payload, privateFrom, privacyGroupId);
+    return post(
+        JSON,
+        sendRequestBesu,
+        "/send",
+        (statusCode, body) -> handleJsonResponse(statusCode, body, SendResponse.class));
+  }
+
+  public ReceiveResponse receive(final String enclaveKey) {
```

Comment:
nit: Not you're problem - but this is badly named too - i.e. maybe the function should be named retrieve?